requirements.txt

```diff
@@ -1,2 +1,3 @@
 Jinja2==2.8
+Django==1.11.4
```

Comment:
I agree with that. Personally I am not fine with installing the whole framework just to use a few things in this framework.
I had initially tried to look into the possibilities of setting up the templates without including the whole Django framework but was unable to do so. So I decided to create the PR as is and get your opinion.